src/wasm-binary.h

```diff
@@ -346,9 +346,10 @@ enum SegmentFlag {
   // Bit 1 if active: 0 = index 0, 1 = index given
   HasIndex = 1 << 1,
   // Table element segments only:
-  // Bit 2: 0 = elemType is funcref and vector of func indexes given
+  // Bit 2: 0 = elemType is funcref and vector of func indexes given (that is,
+  //            this is MVP)
   //        1 = elemType is given and vector of ref expressions is given
-  UsesExpressions = 1 << 2
+  usesExpressions = 1 << 2
```

Comment:
```suggestion
  UsesExpressions = 1 << 2
```

The rest of the enum values are PascalCase, so we should match them (or change them all) for consistency.